ini-file-parser/src/main/java/com/baeldung/inifileparser/IniFileParser.java

```diff
@@ -0,0 +1,19 @@
+package com.baeldung.inifileparser;
+
+import java.io.File;
+
+public class IniFileParser {
+    public static final String FILE_PATH = "C:/Baeldung/sample_config.ini";
```

Comment:
This can't work. The file won't be on this location on the C drive on every reader's machine. Use a relative path. It certainly won't be there on my Mac, where I checkout the repo to `/~/dev/tutorials`

Reply:
Now the file path has to be entered by the user.